state/local_lock_windows.go

```diff
@@ -75,16 +41,8 @@ func (s *LocalState) lock() error {
 }
 
 func (s *LocalState) unlock() error {
-	lockedFilesMu.Lock()
-	defer lockedFilesMu.Unlock()
-
-	handle, ok := lockedFiles[s.stateFileOut]
-	if !ok {
-		// we allow multiple Unlock calls
-		return nil
-	}
-	delete(lockedFiles, s.stateFileOut)
-	return syscall.Close(handle)
+	// the file is closed in Unlock
+	return nil
```

Comment:
Oh yes, file is unocked when the associated handle is closed. This happens automatically when Close is called in the main Unlock method.